src/components/avatar/__tests__/__snapshots__/Avatar.test.tsx.snap

```diff
@@ -7,29 +7,6 @@ Object {
   "asFragment": [Function],
   "baseElement": <body>
```

Comment:
remove this also

Reply:
Since tooltip changes are to be made in avatar we have to update the snaps